app/views/podcast_episodes/show.html.erb

```diff
@@ -50,6 +50,11 @@
       <% else %>
         <h1><%= @episode.title %></h1>
       <% end %>
+      <% if @episode.published_at.present? %>
+        <time class="published-at" datetime="<%= @episode.decorate.published_timestamp %>">
```

Comment:
I think it's worth decorating in the controller action (`StoriesController#handle_podcast_show`) and not decorate multiple times inside a template.